Task error on load

Small issue when run task:

```bash
[webapp] grunt a11y                                                                                                                                                                 
Loading "a11y.js" tasks...ERROR
>> Error: Cannot find module 'lodash'
Loading "a11y.js" tasks...ERROR
>> Error: Cannot find module 'lodash'
Warning: Task "a11y" not found. Use --force to continue.
```

--- a/package.json
+++ b/package.json
@@ -40,10 +40,9 @@
     "gruntplugin"
   ],
   "dependencies": {
-    "a11y": "^0.2.3",
-    "chalk": "^0.5.1",
+    "a11y": "^0.3.0",
+    "chalk": "^1.0.0",
     "indent-string": "^1.2.0",
-    "lodash": "^2.4.1",
     "log-symbols": "^1.0.1",
     "q": "^1.1.2"
   }

--- a/tasks/a11y.js
+++ b/tasks/a11y.js
@@ -10,11 +10,9 @@
 
 var a11y       = require('a11y');
 var chalk      = require('chalk');
-var _          = require('lodash');
 var indent     = require('indent-string');
 var logSymbols = require('log-symbols');
 var Q          = require('q');
-
 
 var fail  = chalk.bold.red;
 var log   = chalk.blue;
@@ -44,7 +42,7 @@ module.exports = function(grunt) {
           if (options.failOnError) {
             grunt.fail.fatal('FATAL: Audit failed for ' + audit.url);
           } else {
-            grunt.log.error('WARN: Audit failed for ' + audit.url)
+            grunt.log.error('WARN: Audit failed for ' + audit.url);
           }
         }
       }).catch(function (error) {

